nifi-nar-bundles/nifi-standard-bundle/nifi-standard-processors/src/main/java/org/apache/nifi/processors/standard/ValidateCsv.java

```diff
@@ -0,0 +1,433 @@
+                return new UniqueHashCode();
+
+            case "strlen":
+                String[] splts = argument.split(",");
+                int[] requiredLengths = new int[splts.length];
+                for(int i = 0; i < splts.length; i++) {
+                    requiredLengths[i] = Integer.parseInt(splts[i]);
+                }
+                return new Strlen(requiredLengths);
+
+            case "strminmax":
+                String[] splits = argument.split(",");
+                return new StrMinMax(Long.parseLong(splits[0]), Long.parseLong(splits[1]));
+
+            case "lminmax":
+                String[] args = argument.split(",");
+                return new LMinMax(Long.parseLong(args[0]), Long.parseLong(args[1]));
+
+            case "dminmax":
+                String[] doubles = argument.split(",");
+                return new DMinMax(Double.parseDouble(doubles[0]), Double.parseDouble(doubles[1]));
+
+            case "equals":
+                if(argument != null && !argument.isEmpty())
+                    throw new IllegalArgumentException("Equals does not expect any argument but has " + argument);
+                return new Equals();
+
+            case "forbidsubstr":
+                String[] forbiddenSubStrings = argument.split(",");
+                for (int i = 0; i < forbiddenSubStrings.length; i++) {
+                    forbiddenSubStrings[i] = forbiddenSubStrings[i].substring(1, forbiddenSubStrings[i].length() - 1);
+                }
+                return new ForbidSubStr(forbiddenSubStrings);
+
+            case "requiresubstr":
+                String[] requiredSubStrings = argument.split(",");
+                for (int i = 0; i < requiredSubStrings.length; i++) {
+                    requiredSubStrings[i] = requiredSubStrings[i].substring(1, requiredSubStrings[i].length() - 1);
+                }
+                return new RequireSubStr(requiredSubStrings);
+
+            case "strnotnullorempty":
+                if(argument != null && !argument.isEmpty())
+                    throw new IllegalArgumentException("StrNotNullOrEmpty does not expect any argument but has " + argument);
+                return new StrNotNullOrEmpty();
+
+            case "requirehashcode":
+                String[] hashs = argument.split(",");
+                int[] hashcodes = new int[hashs.length];
+                for(int i = 0; i < hashs.length; i++) {
+                    hashcodes[i] = Integer.parseInt(hashs[i]);
+                }
+                return new RequireHashCode(hashcodes);
+
+            case "null":
+                if(argument != null && !argument.isEmpty())
+                    throw new IllegalArgumentException("Null does not expect any argument but has " + argument);
+                return null;
+
+            default:
+                throw new IllegalArgumentException(method + " is not an allowed method to define a Cell Processor");
```

Comment:
This should quotes around "method". If a user has two commas in a row it can be quite confusing. 

Also an explicit check if "method" is empty (suggesting the user has two commas in a row) would be nice.